relay-server/src/utils/dynamic_sampling.rs

```diff
@@ -50,21 +50,15 @@ fn check_unsupported_rules(
 
 /// Gets the sampling match result by creating the merged configuration and matching it against
 /// the sampling configuration.
-fn get_sampling_match_result(
+pub fn merge_configs_and_match(
```

Comment:
Do we need this to be public?
```suggestion
fn merge_configs_and_match(
```

Reply:
Thanks!